spec/models/category_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+describe Category do
+  it { should have_many(:videos)}
+
+  it "saves itself" do
+    category = Category.new(name: "comedies")
+    category.save
+    expect(Category.first).to eq(category)
+  end
+
+  it "has many videos" do
+    comedies = Category.create(name: "comedies")
+    south_park = Video.create(title: "South Park", description: "Funny video!", category: comedies) 
+    futurama = Video.create(title: "Furturama", description: "Space travel!", category: comedies) 
+    expect(comedies.videos).to include(futurama, south_park)
+  end
+
+  it "belongs to category" do
```

Comment:
Its better to use should matchers for this. Other than that you are testing Rails default behavior which has been tested by Rails team.